yi-language/yi-language.cabal

```diff
@@ -1,5 +1,5 @@
 name:           yi-language
-version:        0.18.0
+version:        0.18.1
```

Comment:
I thought I commented this before, but there was some previous discussion on the versioning of the split packages: https://github.com/yi-editor/yi/issues/892, they decided to keep all the versions equal because that is more stable. But, I don't really know why they say it is more stable that way, so feel free to ignore it.

Reply:
Aha. I wondered if yi did that but saw that yi-rope is still at 0.11. Is there a reason for that or should I bump everything to 0.19 according to this scheme?

Reply:
yi-rope is basically a separate package. It was split off earlier than all the other packages. It is also used by other packages that are not related to Yi such as [rasa](https://github.com/ChrisPenner/rasa) (mostly just rasa). So, if you want to continue the previous versioning scheme then you should bump everything that is in this repository to 0.19.

Reply:
I've bumped everything to 0.19 to stay with that version scheme